contracts/TiramisuSavingsClub.sol

```diff
@@ -87,13 +91,11 @@ contract TiramisuSavingsClub {
     /// @dev Reverts if you attempt to withdraw more than your saving group balance
     /// @dev Reverts if the caller is not the next payee (not your turn)
     function withdraw(uint _amount) public {
-        require(_amount > 0, "_amount must be > 0");
+        require(_amount > 0, "Withdrawal amount must be positive");
         uint _groupId = getGroupId();
         Group storage _group = groups[_groupId];
-        require(_amount <= _group.balance, "Can't withdraw > current balance");
+        require(_amount <= _group.balance, "Cannot withdraw more than the current balance");
         require(_group.members[_group.nextPayee] == msg.sender, "Caller is not the next payee");
-
-        // solhint-disable-next-line avoid-low-level-calls
         (bool _sent, ) = payable(msg.sender).call{value: _amount}("");
```

Comment:
We should move this external call as the last action in this function, to stop reentrancy attack